environment.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
   - pre-commit=2.6.0
   - pytest=5.4.3
   - pytest-cov=2.10.0
+  - pyyaml=5.4.1
```

Comment:
Did I miss adding this in my PR for the config file?

Reply:
I think so - didn't see it in here. It's already in the environment as a dependency from the other libraries, but good to add list entries for the ones we import directly.